build.proj

```diff
@@ -83,12 +83,12 @@
   </ItemGroup>
   
   <ItemGroup Condition="'$(IncludeBenchmarkData)' == 'true'">
-    <TlcResourceFile Update="@(TlcResourceFile)">
-      <Url>http://aka.ms/tlc-resources/benchmarks/%(Identity)</Url>
+    <BenchmarkFile Update="@(BenchmarkFile)">
+      <Url>https://tlcresources.blob.core.windows.net/mslrweb10k/%(Identity)</Url>
```

Comment:
Has there been a consensus reached to not use an aka.ms link? Otherwise I'd keep w/ the current aka.ms links. 

The aka.ms links allows us offer backwards comparability to later alter the download path for existing versions of ML.Net & ML models in production. For instance, if we later wanted to serve the content through another CDN/location like Akamai/CloudFront or (GitHub LFS + https://dotnet.github.io/machinelearning/) we can later change the aka.ms link to point to the new locations without maintaining all previous versions.

Reply:
In addition, we should use https for security (instead of http) for the aka.ms links.

Reply:
I agree with Justin here. Why did this change from `aka.ms`?  My previous comment was referring to having `tlc` in the name of the URL. My thoughts are that we should have `[https://aka.ms/mlnet-resources`](https://aka.ms/mlnet-resources`) or similar and not have `tlc` in the URL.

---
In reply to: [217588750](https://github.com/dotnet/machinelearning/pull/888#discussion_r217588750) [](ancestors = 217588750)

Reply:
also, the aka.ms was the CDN link. This is the direct blob link. 

---
In reply to: [217748910](https://github.com/dotnet/machinelearning/pull/888#discussion_r217748910) [](ancestors = 217748910,217588750)

Reply:
> Why did this change from aka.ms? 

these were the links in the onenote. i didnt know that they are direct links, and we can access them through aka.ms

Reply:
@justinormont from where can i get the aka.ms links ? i tried http://aka.ms/tlc-resources/mslrweb10k/%(Identity) but thats not working 

cc @artidoro